src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -1287,8 +1397,8 @@ CREATE TABLE transient_object_directory
 CREATE TABLE user_account
 (usr_id                      INTEGER                       NOT NULL
 ,dm_user_s_object_id         CHARACTER VARYING(16)
-,user_name                   CHARACTER VARYING             NOT NULL
-,user_full_name              CHARACTER VARYING             NOT NULL
+,user_name                   CHARACTER VARYING            
+,user_full_name              CHARACTER VARYING            

```

Comment:
Still finding it tricky to understand the identifying attributes on this table, if more columns are nullable.
Wondering what is important for the user state login information - are we still expecting user_email_address in modernised?

Reply:
I believe the intention here is to relax the not-null constraints to aid migration, but with the intention of tightening up again upon completion.
For example IMO user_name is the principal internal user_name and email is a natural key for user_account , yet both are nullable, to accommodate any intermediate unknowns.